tests/python/ci/test_ci.py

```diff
@@ -33,6 +34,168 @@ def parameterize_named(*values):
     return pytest.mark.parametrize(",".join(keys), [tuple(d.values()) for d in values])
 
 
+@tvm.testing.skip_if_wheel_test
+@pytest.mark.parametrize(
+    # pylint: disable=line-too-long
+    "main_xml_file,main_xml_content,pr_xml_file,pr_xml_content,target_url,s3_prefix,"
```

Comment:
also the test names here that pytest autogenerates and shows in test reports are crazy long since they include the whole junit string, could you also pass through `ids=[...]` with a short name for each case? i.e. something similar to https://github.com/apache/tvm/blob/main/tests/python/ci/test_tvmbot.py#L141-L145

Reply:
Thanks! Seems like a test was also flaky due to different order of reported skipped tests. That should be sorted now.